public/apps/account/test/__snapshots__/log-out-button.test.tsx.snap

```diff
@@ -1,11 +1,63 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Account menu - Log out button renders renders when auth type is MultiAuth: basicauth 1`] = `
```

Comment:
nit : typo for `renders`

Reply:
This is not typo.

Reply:
This looks like an existing type that is not required to be fixed in this PR, right? But is it feasible to fix it in another PR?

Reply:
I offline connected with @aoguan1990. Thanks for pointing me to [the place](https://github.com/opensearch-project/security-dashboards-plugin/blob/main/public/apps/account/test/log-out-button.test.tsx#L37-L38) where the message gets generated!